vale/styles/jargonLint/README.md

```diff
@@ -0,0 +1,9 @@
+# jargonList rules
+
+## Tooling
+
+substitution can be sorted with this
```

Comment:
Something like: "To sort rules that extend substitution:" might be better.

Instead of extend, you could also use apply or use as well